openedx/features/enterprise_support/apps.py

```diff
@@ -0,0 +1,16 @@
+"""
+Configuration for enterprise_support
+"""
+from django.apps import AppConfig
+
+
+class EnterpriseSupportConfig(AppConfig):
+    """
+    Configuration class for enterprise_support
+    """
+    name = 'openedx.features.enterprise_support'
+
+    def ready(self):
+        # Import signals to activate signal handler which update
+        # the SailThru variables every time an enterprise learner is created.
```

Comment:
No need to mention `SailThru` here. We could add signals for other purposes in the future.